src/components/index.js

```diff
@@ -0,0 +1,5 @@
+export { Navbar } from "./Navbar";
```

Comment:
suggestion: you can short this line by `export * from  "./Navbar" ` it exports everything from that file

Reply:
Sure will make this change.